extension/export_OneParticleOperator.cc

```diff
@@ -112,6 +114,8 @@ void export_OneParticleOperator(py::module& m) {
              "Set a block of the matrix (e.g. o1o1, o1v1)")
         .def("set_zero_block", set_zero_block_1,
              "Set a block of the matrix (e.g. o1o1, o1v1) to be explicitly zero.")
+        .def("set_zero_block", set_zero_block_2,
```

Comment:
I'd prefer `py::overload_cast` if possible.

```C++
.def("set_zero_block", py::overload_cast<std::vector<std::string>>(&OneParticleOperator::set_zero_block),
"Set a few blocks of the matrix (e.g. o1o1, o1v1) to be explicitly zero.")
```
Then it's all in one place.

[Link](https://pybind11.readthedocs.io/en/stable/classes.html?highlight=overload_cast#overloaded-methods)

Reply:
Ok, I'll try that. With some of the pybind features it tends to give very long lines, which destroys the overview a little (that's why I rarely use the lambdas), but this might work.

Reply:
I guess if one breaks the line after the first argument, it should be okay.